cluster-autoscaler/processors/status/scale_down_status_processor.go

```diff
@@ -119,6 +119,8 @@ const (
 	NodeDeleteErrorFailedToDelete
 	// NodeDeleteErrorInternal - failed to delete the node because of an unexpected error.
 	NodeDeleteErrorInternal
+	// NodeDeleteInPogress - node deletion is currently in progress.
```

Comment:
Hmm not sure if we should introduce a new type result for in progress - these values are supposed to be consumed by the scale down status processor, but only for nodes that are actually deleted successfully, or if the deletion fails. Introducing an "in progress" result could be confusing for processor implementations - how should they handle it, will the node be included in another event with a different result? I know it won't happen with the current Actuator implementation, but processor implementations shouldn't have to care about the implementation details of Actuator, so they're not necessarily aware of that.

So I'd either just use `NodeDeleteOk` for the intermediate result, or introduce this as a _private_ const in the actuation package, as it's an implementation detail of that package.

Reply:
Ack